src/components/Search.js

```diff
@@ -1,9 +1,57 @@
-import React from 'react';
- 
-
-export const Search = () => (
-    <div className="c-page">
-        <h1>Pomoc</h1>
-        <img src="https://via.placeholder.com/250x400" alt="placeholder"></img>
-    </div>
-);
\ No newline at end of file
+// import React, { Component } from "react";
+
+// export class Search extends Component {
+//   constructor(props) {
+//     super(props);
+//     this.state = {
+//       filtered: []
+//     };
+//   }
+
+//   componentDidMount() {
+//     this.setState({
+//       filtered: this.props.plants
+//     });
+//   }
+
+//   componentWillReceiveProps(nextProps) {
+//     this.setState({
+//       filtered: nextProps.plants
+//     });
+//   }
+
+//   handleSearch = e => {
+//     let newList = [];
+
+//     if (e.target.value !== "") {
+//       newList = this.state.currentList.filter(plant => {
+//         const lc = plant.toLowerCase();
+//         const filter = e.target.value.toLowerCase();
+//         return lc.icludes(filter);
+//       });
+//     } else {
+//       newList = this.state.currentList;
+//     }
+//     this.setState({
+//       filtered: newList
+//     });
+//   };
+
+//   render() {
+//     return (
+//       <div>
+//         <input
+//           type="text"
+//           className="input"
+//           placeholder="Search..."
+//           onChange={this.handleSearch}
+//         />
+//         <ul>
+//           {this.state.filtered.map(plant => (
```

Comment:
This will explode because in place where you use `<Search />` you do not pass any props yet. So this part of code will try to run map on undefined

Reply:
This component is removed for there is no use for it anymore.

Reply:
Ok, then we can get rid of this file, right? :) Or do you plan to use this code?